integrated_channels/integrated_channel/tasks.py

```diff
@@ -119,13 +119,19 @@ def remove_duplicate_transmission_audits():
         duplicates_found += num_duplicates
         # Mysql doesn't support taking the count of a sliced queryset
         duplicates_to_delete = duplicates[1:]
-        if getattr(settings, "DISABLE_REMOVE_DUP_TRANSMISSION_AUDIT_DRY_RUN", False):
-            for duplicate in duplicates_to_delete:
-                duplicate.delete()
-        else:
+        LOGGER.info(
+            f"remove_duplicate_transmission_audits task dry run mode" +
+            f"set to: {getattr(settings, 'DRY_RUN_MODE_REMOVE_DUP_TRANSMISSION_AUDIT')}"
+        )
+        if getattr(settings, "DRY_RUN_MODE_REMOVE_DUP_TRANSMISSION_AUDIT", True):
             LOGGER.info(
                 f"Found {num_duplicates} duplicate content transmission audits for course: {content_id}"
             )
+        else:
+            LOGGER.info(f'Beginning to delete duplicate content transmission audits for course: {content_id}')
+            for duplicate in duplicates_to_delete:
+                LOGGER.info(f"Deleting duplicate transmission audit: {duplicate.id}")
```

Comment:
yeeeah, I was between the two ¯\_(ツ)_/¯

Reply:
imho i prefer `deleting...` (logging before) because if the delete blows up, the next lines would be a stack trace and you'd know which record it blew up on more easily.